samples/connectedhomeip/light_bulb/src/LightBulbPublishService.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "LightBulbPublishService.h"
+
+#include "AppTask.h"
+
+#include <logging/log.h>
+#include <zephyr.h>
+
+#include <inet/InetConfig.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <transport/TransportMgr.h>
+#include <transport/raw/UDP.h>
+
+LOG_MODULE_DECLARE(app);
+
+const char kPublishServicePayload[] = "chip-light-bulb";
+const char kPublishDestAddr[] = "ff02::1";
```

Comment:
Can we use `ff03::1` instead? Since `ff02::1` will work only in a topology where Ligh Bulb is directly connected to Light Switch. While i can imagine different topology e.g. with Border Router in between.

Reply:
I commented that remark in the different PR: https://github.com/kkasperczyk-no/sdk-nrf/pull/4#discussion_r561644073